hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/ContainerKeyMapperTask.java

```diff
@@ -90,18 +91,24 @@ public Pair<String, Boolean> reprocess(OMMetadataManager omMetadataManager) {
       reconContainerMetadataManager
               .reinitWithNewContainerDataFromOm(new HashMap<>());
 
-      Table<String, OmKeyInfo> omKeyInfoTable =
-          omMetadataManager.getKeyTable(getBucketLayout());
-      try (TableIterator<String, ? extends Table.KeyValue<String, OmKeyInfo>>
-               keyIter = omKeyInfoTable.iterator()) {
-        while (keyIter.hasNext()) {
-          Table.KeyValue<String, OmKeyInfo> kv = keyIter.next();
-          OmKeyInfo omKeyInfo = kv.getValue();
-          handlePutOMKeyEvent(kv.getKey(), omKeyInfo, containerKeyMap,
-              containerKeyCountMap, deletedKeyCountList);
-          omKeyCount++;
+      // loop over both key table and file table
+      for (BucketLayout layout : Arrays.asList(BucketLayout.DEFAULT,
```

Comment:
why did you choose to use `BucketLayout.DEFAULT` here instead of `BucketLayout.LEGACY` (like in `ContainerEndpoint` on [line 177](https://github.com/apache/ozone/blob/f6f42207ba0a6bf18a58145d352c4904250a32ef/hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/ContainerEndpoint.java#L177))? later the default bucket layout could change (I believe). I think we should use `BucketLayout.LEGACY` consistently (I am fine with `BucketLayout.DEFAULT` too, but it should be consistent)

Reply:
Actually, I think LEGACY is much better than DEFAULT.  I believe the DEFAULT has changed a number of times already and when it does, that may invalidate that are using it as a synomym for LEGACY